cmd/kubeadm/app/util/config/masterconfig_test.go

```diff
@@ -30,12 +30,13 @@ import (
 )
 
 const (
-	master_v1alpha2YAML   = "testdata/conversion/master/v1alpha2.yaml"
-	master_v1alpha3YAML   = "testdata/conversion/master/v1alpha3.yaml"
-	master_internalYAML   = "testdata/conversion/master/internal.yaml"
-	master_incompleteYAML = "testdata/defaulting/master/incomplete.yaml"
-	master_defaultedYAML  = "testdata/defaulting/master/defaulted.yaml"
-	master_invalidYAML    = "testdata/validation/invalid_mastercfg.yaml"
+	master_v1alpha2YAML       = "testdata/conversion/master/v1alpha2.yaml"
+	master_v1alpha3YAML       = "testdata/conversion/master/v1alpha3.yaml"
+	master_internalYAML       = "testdata/conversion/master/internal.yaml"
+	master_incompleteYAML     = "testdata/defaulting/master/incomplete.yaml"
+	master_defaultedYAML      = "testdata/defaulting/master/defaulted.yaml"
+	master_forceDefaultedYAML = "testdata/defaulting/master/defaulted-forced.yaml"
```

Comment:
`s/defaulted-forced/defaulted_forced` 
the old scheme seems to use files with `_`.